src/components/elements/text/text.tsx

```diff
@@ -94,6 +95,7 @@ export const Text = (props: IGenericElementProps<ITextSchema>) => {
 				ref={elementRef}
 				maxLines={!expanded ? maxLines : undefined}
 				data-testid={getTestId(id)}
+				weight={_.isNumber(weight) ? WEIGHT_MAPPING[weight] : weight || undefined}
```

Comment:
the fallback for weight seems unnecessary

```suggestion
				weight={_.isNumber(weight) ? WEIGHT_MAPPING[weight] : weight}
```